godtools/XMLManagement/XMLManager.swift

```diff
@@ -26,7 +26,7 @@ class XMLManager: NSObject {
             properties[attribute.name] = attribute.text
         }
         
-        if data.element?.text != nil && data.element?.text?.trimmingCharacters(in: .whitespaces) != "" {
+        if data.element?.text != nil && data.element?.text.trimmingCharacters(in: .whitespaces) != "" {
```

Comment:
Why was the `?` removed?

Reply:
Same as the other comment. This was a required as part of building in XCode 9. I'm not sure why, but the build would not pass w/o this change.

Reply:
Ok, thanks!